### DISCUSSION TITLE
Has anyone updated nanovdb_viewer to latest version of NanoVDB?

### DISCUSSION THREAD

**Discussion by GrouchoChaplin** [AcademySoftwareFoundation/openvdb#1636] [Q&A]
Has anyone updated nanovdb_viewer to latest version of NanoVDB?  I am trying to compile the nanovdb_viewer tool under the latest OpenVDB, but there have been changes to NanoVDB since the last branch in which nanovdb_viewer appeared.

**Comment by trifonov-borislav** [comment]
It's been several years since the viewer was removed. What's the plan for bringing it back?
